test/test_tables.py

```diff
@@ -168,6 +168,31 @@ def test_select():
     assert have.y[0].as_py() == 6
 
 
```

Comment:
Please add tests, at least for `column`, of the following cases:
1. The child is null, while the parent has non-null data in other rows (`Wrapper.from_kwargs(id=["1", "2"], pair=[None, None])`).
2. The table is empty (`Wrapper.empty()`)
3. The dot-delimited syntax specifies a column that does not exist (`wrapper.column("unknown.whatever")`)

For `select in particular:
1. If wrapper has an attribute, it should be preserved through the select call. (I think this is broken in HEAD)
2. If child has an attribute, it should be preserved through the select call.